snowballDevs/kandoo
# Basic passport implementation

## Description

Implemented basic auth with local passport strategy which lets user login using their email and password
-  still need to add discord & github strategies in future if we decide to allow users to login using their respective accounts

## Related Issue

-   [x] Is this related to a specific issue? Is so please specify: #10 

## Type of Changes

Use a check for ✓ the following type of changes:

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

## Testing Steps / QA Criteria

Tested with Chris, using postman to sign up a user, then to login a user to see if they're authenticated

## Checklist:
